apps/meteor/app/livechat/server/hooks/offlineMessageToChannel.ts

```diff
@@ -29,8 +29,8 @@ callbacks.add(
 			return data;
 		}
 
-		const room = await Rooms.findOneByName(channelName, { projection: { t: 1, archived: 1 } });
-		if (!room || room.archived || room.closedAt) {
+		const room: any = await Rooms.findOneByName(channelName, { projection: { t: 1, archived: 1 } });
+		if (!room || room.archived || room?.closedAt) {
```

Comment:
```suggestion
		const room = await Rooms.findOneByName(channelName, { projection: { t: 1, archived: 1 } });
		if (!room || room.archived || (isOmnichannelRoom(room) && room.closedAt)) {
```
The `closedAt` porperty is part of the `IOmnichannelGenericRoom` type.
Also add this import: `import { isOmnichannelRoom } from '@rocket.chat/core-typings';`